app/models/proposal.rb

```diff
@@ -60,10 +60,7 @@ class Proposal < ActiveRecord::Base
   scope :for_state, ->(state) do
     where(state: state).order(:title).includes(:event, {speakers: :user}, :review_taggings)
   end
-  scope :in_track, ->(track) do
-    track = nil if track.try(:strip).blank?
-    where(track: track)
-  end
```

Comment:
This wasn't used much, and there wasn't a spec around it. I wanted to leverage it for the new stats, and I convinced myself it wasn't working as expected. `track.try(:strip).blank?` resolved to true when passing a track instance, track name, or track id. With the changes, one can retrieve proposals in 'General'/no-track by passing `nil`.